db/migrate/20220915115442_createlikes.rb

```diff
@@ -0,0 +1,4 @@
+class Createlikes < ActiveRecord::Migration[7.0]
+  def change
+  end
+end
```

Comment:
- [ ] When I am trying to run the migrations, I am getting errors, one of the errors say `relation likes does not exist`, which is true, because  create likes migration is not creating a likes table table, please add a block to create likes table inside : 

```
  create_table :likes do |t|
    t.string :title

    t.timestamps
  end
```